src/item_factory.cpp

```diff
@@ -1109,6 +1113,22 @@ void Item_factory::load_wheel( JsonObject &jo, const std::string &src )
     }
 }
 
+void Item_factory::load( islot_fuel &slot, JsonObject &jo, const std::string &src )
+{
+    bool strict = src == "core";
```

Comment:
A long-term solution would be to have `src` be a mod data structure rather than dumb string. That way we could designate mods as strict.